kola/tests/ignition/kernel.go

```diff
@@ -0,0 +1,39 @@
+// Copyright The Mantle Authors
+// SPDX-License-Identifier: Apache-2.0
+package ignition
+
+import (
+	"github.com/flatcar-linux/mantle/kola/cluster"
+	"github.com/flatcar-linux/mantle/kola/register"
+	"github.com/flatcar-linux/mantle/platform/conf"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Name:        "cl.ignition.kargs",
+		Run:         check,
+		ClusterSize: 1,
+		UserData: conf.Ignition(`{
+		  "ignition": {
+		    "version": "3.3.0"
+		  },
+		  "storage": {
+		    "files": [
+		      {
+			"path": "/works",
+			"contents": {
+			  "source": "data:,helloworld%0A",
+			  "verification": {}
+			},
+			"mode": 420
+		      }
+		    ]
+		  },
+		  "kernelArguments": {
+		    "shouldExist": ["flatcar.first_boot=detected"]
```

Comment:
```suggestion
		    "shouldExist": ["flatcar.first_boot=detected"]
```
I would use something like `flatcar.autologin` because it is harmless compared to forcing to run ignition this way which can maybe cause a reboot to fail due to not being able to remove the flag file

Reply:
Or maybe more generic like `quiet` if you want to upstream it

Reply:
but from what I understood, it's up to the distro maintainer to provide the helper: `ignition-kargs-helper`

so not sure if it's something upstreamable.

Reply:
I think Fedora CoreOS has this, so the coreos-assembler repo's kola would be able to run it